index.html

```diff
@@ -8,6 +8,7 @@
     />
     <title>Rick and Morty Character Log</title>
     <link rel="stylesheet" href="./styles.css" />
+    <script src="./index.js" type="module"></script>
```

Comment:
There's already a `<script>` html tag at `line 66` (I also did the same thing initially because I didn't notice it haha) just remove one of the two :)